src/core_plugins/kibana/public/dashboard/panel/panel.js

```diff
@@ -117,6 +117,10 @@ uiModules
           $scope.$watchCollection('panel.sort', function () {
             $scope.saveState();
           });
+
+          $scope.setSortOrder = function setSortOrder(columnName, direction) {
```

Comment:
Yes, that's a good idea. Looks like that's the only place where the sorting can be changed. It wouldn't just improve readability but also performance (fewer evaluations during digest). Thank you!